web/modules/custom/dpl_event/src/Services/EventRestMapper.php

```diff
@@ -83,6 +85,51 @@ public function getResponse(EventInstance $event_instance): EventsGET200Response
     ]);
   }
 
+  /**
+   * Getting associated branches.
+   *
+   * @return ?string[]
+   *  The translated branch labels.
+   */
+  private function getBranches(): ?array {
```

Comment:
Consider why we would want to return `NULL` here.

As I read the code an empty array would work just as well and simplify the code.